src/modules/fancyzones/tests/UnitTests/JsonHelpers.Tests.cpp

```diff
@@ -1763,6 +1763,36 @@ namespace FancyZonesUnitTests
                 compareJsonObjects(expectedJsonObj, actualJson);
             }
 
+            TEST_METHOD (SaveFancyZonesDataWithFastAccessLayoutKeys)
```

Comment:
The test fails because of the logger

Reply:
We have this scheduled for 0.35 https://github.com/microsoft/PowerToys/issues/9684